### ISSUE TITLE
Many warnings about this

### ISSUE THREAD

**Issue by daogangtang** [libp2p/rust-libp2p#739]
like this.
```
warning: use of deprecated item 'tokio_io::codec::length_delimited': Moved to tokio-codec
  --> protocols/secio/src/handshake.rs:46:5
   |
46 | use tokio_io::codec::length_delimited;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(deprecated)] on by default
```

**Comment by dvdplm** [comment]
Yes, it's a bit frustrating. The issue is [here](https://github.com/tokio-rs/tokio/issues/680) and a PR to fix it [here](https://github.com/tokio-rs/tokio/pull/685).

**Comment by tomaka** [comment]
This has been fixed through #1328
